js/tab.js

```diff
@@ -20,11 +20,15 @@ var FlatUIColors = [
 
 function Tab (opts) {
   var host = opts.host.replace(/\./g, "-");
+  var channel = opts.chan;
```

Comment:
there is no need for this variable if you just assign `card.dataset.channel = opts.channel;`